src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $item-gap;
+  width: ($item-size * 8) + ($item-gap * 6);
```

Comment:
You have only 7 days in 1 row, so multiply $item-size to 7. And don't forget about property "box-sizing: border-box". And apply it to all elements on page (use selector *).